lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -223,7 +223,7 @@ func (handler *ContainersHandlersImpl) CommitHandler(params containers.CommitPar
 	}
 
 	if err := h.Commit(context.Background(), handler.handlerCtx.Session, params.WaitTime); err != nil {
-		log.Errorf("CommitHandler error on handle(%s) for %s: %#v", h.String(), h.ExecConfig.ID, err)
+		log.Errorf("CommitHandler error on handle(%s) for %s: %s", h.String(), h.ExecConfig.ID, err)
```

Comment:
No need to call String for stringers. %s should do the job.

Reply:
It is already %s?

Reply:
Ahh the argument is still calling String(), will fix in a sec.